### PR TITLE
Updates for Arch, Fedora and RHEL

### PR DESCRIPTION
**Pull request by ne9z**
Clean up unsupported boot environment manager and upgrade RHEL to major version 9.

### CODE REVIEW

**Comment by ne9z** [comment]
@gmelikov

**Comment by gmelikov** [comment]
@ne9z feel free to ping me when it's ready, looks like you've added changes since last mention.

**Comment by ne9z** [comment]
@gmelikov That's very considerate of you, thanks!  I've squashed the commits and they are ready for merge now.

By the way, I've added Root on ZFS support for Alpine Linux in another pull request.

**Comment by ne9z** [comment]
@gmelikov Hello, I think both this PR and #331 are ready for merge now.

File: docs/Getting Started/NixOS/Root on ZFS/2-system-installation.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
+.. highlight:: sh
```
Comment (gmelikov) [inline docs/Getting Started/NixOS/Root on ZFS/2-system-installation.rst:1]: Minor note for future - file rename will lead to different link, if it's possible - we should leave same links to minimize 404s on external links to our docs.

**Review by gmelikov — APPROVED** [review]
Thank you, will merge as is.

**Comment by cbc02009** [comment]
Just out of curiosity, why was the UUID removed from arch linux? Did something change that made it no longer necessary?

Why were the commands to create a subvolume for the user's home removed?

**Comment by ne9z** [comment]
@cbc02009

UUID was removed as part of an effort to simplify the guide:  the guide was becoming more and more bloated and the commands ugly to look at.

The new principle is to do a bare minimum of configuration here at this repo and refer users to wiki articles where necessary.

Datasets for /home is brought back by a pending pull request: #333 @gmelikov Can you review it?

**Pull request merged**